src/ChatPage.cc

```diff
@@ -575,8 +576,12 @@ ChatPage::showQuickSwitcher()
 
         QMap<QString, QString> rooms;
 
-        for (auto it = state_manager_.constBegin(); it != state_manager_.constEnd(); ++it)
-                rooms.insert(it.value().getName(), it.key());
+        for (auto it = state_manager_.constBegin(); it != state_manager_.constEnd(); ++it) {
+                QString deambiguator = it.value().canonical_alias.content().alias();
```

Comment:
That's ok for now but eventually we will have to show the room list in a more custom way.

Reply:
Yes of course. I just wanted to ensure that all rooms get listed as is.